app/views/machines/show.html.erb

```diff
@@ -1 +1,8 @@
 <h1><%= @machine.location %> Vending Machine</h1>
+
+<section class="machine-snacks-main-block" >
+  <%= @machine.snacks.each do |snack| %>
+    <h3><%= snack.name.upcase %>: $<%= snack.price %></h3>
```

Comment:
you can look into an action view helper called `number_to_currency` to help format these prices